packages/gatsby-remark-katex/package.json

```diff
@@ -28,8 +28,8 @@
   "license": "MIT",
   "main": "index.js",
   "peerDependencies": {
-    "gatsby": ">2.0.0-alpha",
-    "katex": ">=0.10.0 <1.0.0"
+    "gatsby": "^2.0.0",
+    "katex": "^0.10.0"
```

Comment:
Excellent point!

Reply:
but @wardpeet both are same.

Reply:
@sibiraj-s they aren't when it's below version 1.0.0 😄 
you can find out why here https://nodesource.com/blog/semver-tilde-and-caret/

Reply:
Just one more thing to ask

> Major version zero (0.y.z) is for initial development. Anything may change at any time.

based on the above quote. in that case won't `>=0.10.0 <1.0.0` this will be more problematic too. Can we pin this version to 0.10 🤔

Reply:
`^0.10.0` seems fine here - it won't allow `^0.11.0` (well users, can install it given it's peerDep, but it will complain for anything >=0.11.0)